git-log-develop-from-dec7.txt

```diff
@@ -0,0 +1,414 @@
+BRIAN - DONE commit ccdb51175eec7c775425a288e5b62b355480b614
+Author: Jerry Wang <58975336+wangzixi-diablo@users.noreply.github.com>
```

Comment:
For Jerry's PR: 
What do you think, should we carry over the spinner?
The reason why I'm asking is what if Jalo fails after the retries? It's not likely to happen, but still technically possible if back-end is overloaded, or something.

Reply:
1 - 🙈  develop delivery mode step 'styling' is broken due to his change
2 - was not a spinner. He disabled the inputs and continue button. Current fix that I just did utilizes the spinner upon selecting a delivery mode and disables the continue button as well. (Need to talk and normalize what way we go because most of the time, we just use spinners instead of disabling the 'inputs')

Reply:
Is it broken in the new checkout? If not, no need to "fix" develop, as the new checkout will be the default one (and possibly the only one).
Ideally, we want to use the progress button + disable the form while loading. But this is the task for the future, where we will refactor components.
Let's just use the spinner. Optionally, we can disable the form if it's easy enough to do so, or if it makes sense.

Reply:
conclusion that we came up with:

- will discuss with UI/UX starting January on new checkout behavior in terms of loading 'spinners' or 'disabling + spinners'